packages/wonder-blocks-i18n/src/functions/i18n.ts

```diff
@@ -260,12 +261,23 @@ export const ngettext: ngettextOverloads = (
 
     // Get the options to substitute into the string.
     // We automatically add in the 'magic' option-variable 'num'.
-    actualOptions.num = actualOptions.num || actualNum;
+    actualOptions.num = formatNumber(actualNum);
```

Comment:
**question:** I am confused by how this change relates to custom formatting - perhaps it's down to the test examples only using numbers and not strings. Basically, the `num` option could override what number was actually getting formatted. Is this being used in webapp code to pass a string instead of a number, and therefore do custom formatting?

I'm all for removing the extra confusing option - I just want to understand how that impacts formatting.

Reply:
Yes, it was being used to pass in a string - there were test cases for this here in wonder-blocks-i18n that were testing this case, too, but I removed them as part of this change. So now there will be no way to have a custom format of the string, it'll just be formatted with `Intl.NumberFormat` instead.

Reply:
Yeah, I saw the deleted test cases; it's just they always used a primitive number so I didn't get that you could just pass a string. Thanks for clarifying.